### PR TITLE
Remove login button and supporting oauth code

### LINKED ISSUE #805: Remove login button and supporting software from web app
With the implementation of the SSO layer on top of the dev, test, and ops web apps, I don't think there is a current need to have and maintain our own authentication solution (i.e. the 'login' button and the software involved with it).

### PR DESCRIPTION
**Pull request by bourque**
Closes #805 

This PR removes the `login` button on the homepage, as well as all of the authentication code (e.g. `oauth.py`) that was needed to support the login feature.  The login feature is no longer needed as users are now authenticated through a SSO layer attached to an AD group (see https://innerspace.stsci.edu/display/JWQLPROJ/Authentication+Overview).

One downside to this is now we do not have a way to identify who is flagging anomalies.  For now, I switched the `user` value in the anomaly database table to store a value of `unknown` until we can find a workaround for this.  Perhaps there is a way to gather user information from the SSO/AD layer, but I will need to consult with ITSD about that.

### CODE REVIEW

File: jwql/website/apps/jwql/templates/base.html [hunk 1]
```diff
@@ -178,20 +147,20 @@
...
+            	<li class="nav-item active d-none d-xl-block">
+					<a id="github-link"
+					   class="nav-link"
+					   href="https://github.com/spacetelescope/jwql"
+					   target="_blank">
+						<img src={{ static('img/githublogo.svg') }}
+							 height=35
+							 width=35
+							 alt="GitHub Logo"
+							 title="GitHub Logo">
```
Comment (bourque) [inline jwql/website/apps/jwql/templates/base.html:161]: This bit is to just put the GitHub icon on the right-hand side of the navbar, where the login button used to be.

**Comment by bourque** [comment]
@tnking97 Since this is the last PR needed to get in before the `v1.0` release, would you be able to review this before you start the release process?

**Comment by tnking97** [comment]
Hi @bourque! Yes, I'm happy to review it!

**Comment by bourque** [comment]
Awesome, thank you @tnking97!

**Comment by tnking97** [comment]
Hey @bourque, when I run this locally, there is still a login button, and when I try to login I get the following error: `{"detail":{"error":"invalid_client","error_description":""}}`. Should the GitHub icon replace the login button locally as well, or is there something I'm missing? I'm not getting to the SSO page.

**Comment by bourque** [comment]
@tnking97 Oh interesting, that is odd. I am assuming you pulled/checked out the branch in the usual way?  The login button should be gone completely now (and the GitHub icon should now be on the right-hand-side of the navbar).  I'm not sure why you would still be seeing the login button 🤔 

It makes sense that you will not see the SSO page, since this is being run locally.  The SSO/AD authentication is only set up for the dev, test, and (soon-to-be) ops servers.

File: jwql/website/apps/jwql/templates/base.html (hunk 1 above)
Comment (tnking97) [reply jwql/website/apps/jwql/templates/base.html:161]: It is nice to have the GitHub icon here!

File: jwql/website/apps/jwql/views.py [hunk 2]
```diff
@@ -767,17 +753,11 @@ def view_image(request, user, inst, file_root, rewrite=False):
...
-    if '127.0.0.1' in get_base_url():
-        if user['ezid'] is None:
-            user['ezid'] = 'dev'
-
     # If this is a POST request, process the form data
     if request.method == 'POST':
         anomaly_choices = dict(request.POST)['anomaly_choices']
         if form.is_valid():
-            form.update_anomaly_table(file_root, user['ezid'], anomaly_choices)
+            form.update_anomaly_table(file_root, 'unknown', anomaly_choices)
```
Comment (tnking97) [inline jwql/website/apps/jwql/views.py:760]: The anomaly table update feature seems to still work smoothly.

**Review by tnking97 — APPROVED** [review]
This PR looks good and is running smoothly.

**Comment by bourque** [comment]
Thank you @tnking97!

**Pull request merged**